File: Krootify.Domain/Song.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Song : BaseEntity
    {
        public string Name { get; set; }
        public string SongUrl { get; set; }
        public string PhotoUrl { get; set; }
        public DateTime ReleaseDate { get; set; }
        public int ArtistId { get; set; }
        public Artist Artists { get; set; }
        public int AlbumId { get; set; }
        public Album Albums { get; set; }
        public ICollection<PlayListSong> PlayLists { get; set; }
        public TimeSpan Duration { get; set; }
        public int Listeners { get; set; }
        public int SongGenreId { get; set; }
        public SongGenre SongGenres { get; set; }
        public int SongCategoryId { get; set; }
        public SongCategory SongCategories { get; set; }
    }
}

File: Krootify.Angular/src/app/_models/Song/song.ts
export interface ISong {
    id:           number;
    count:        number;
    name:         string;
    songUrl:      string;
    photoUrl:     string;
    releaseDate:  Date;
    artists:      string;
    albums:       string;
    duration:     number;
    listeners:    number;
    songGenre:    string;
    songCategory: string;
    songGenreId:    number;
    songCategoryId: number;
    artistId:      number;
    albumId:       number;
    dateAdded: string;
}